templates/main.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
consider using [base and child](https://jinja.palletsprojects.com/en/2.11.x/templates/#base-template) templates to keep things that would be common to all pages in one place

Reply:
I try to understand, but I have some problem in the contact, you want us to use only blocking 
 and not include?  I separate the header's and the navgar to separate css's.
the contact of the navgar and the footer should me out side from the include? even if we used then in all our pages?

Reply:
there is also the <head> section, the <html> and <body> tags and the <script> tags - all of those will probably repeat on all pages.

Consider how much stuff you have in this file right now considering that it does not yet contain any content...

I recommend using the inheritance mechanisms of jinja since they are more flexible the simple includes.

Reply:
Ok, thanks 
I hope now it more lite base and child, I still you include but I create a layout html that the home page used it